### ISSUE TITLE
Woody Woodpecker (GBC) - Buzzing noise in Woody Woodpecker logo

### ISSUE THREAD

**Issue by AngrySonics** [mgba-emu/mgba#3335]
Systems tested: Windows 11 23H2
mGBA versions tested: 0.10.3, 0.11-8592-77e747ebf/2024-11-04-win64-8592-77e747ebf6e2becd669312667999e39e79036ceb, 0.10.5
ROM: USA

When starting the game up and the Woody Woodpecker laugh sound plays, it starts making a buzzing sound in the background that ends as soon as the laugh stops. I'm highly confident this is not a ROM issue, as it does not buzz when running in SameBoy.

https://github.com/user-attachments/assets/5a8979b3-9231-4627-82bd-37340e134de8

**Comment by RafaelOliveira71594** [comment]
It May Look Like An Issue With mGBA's Sound Settings Instead Of The ROM Since It Works Fine On Sameboy Did You Try Resetting Your Settings To Default ?

**Comment by AngrySonics** [comment]
The buzzing is still there even after I reset mGBA's settings.

**Comment by endrift** [comment]
It's definitely not a settings issue.

**Comment by RafaelOliveira71594** [comment]
Okay It May Be That Game Isn't Compatible With mGBA Or Something l Have The Same Bug

**Comment by endrift** [comment]
Yes that's called a bug. That's why this, a bug report, exists. So it can get fixed.

**Comment by RafaelOliveira71594** [comment]
> Yes that's called a bug. That's why this, a bug report, exists. So it can get fixed.

Thanks l Will Use Bug Report You Too AngrySonics

**Comment by endrift** [comment]
Fortunately I've already gathered enough information that I don't need the bug report zip this time. It appears to be an issue with audio channel 3. I'll take a closer look after I've released 0.11.

**Comment by RafaelOliveira71594** [comment]
l Have The Correct ROM That Is USA

**Comment by RafaelOliveira71594** [comment]
My Bug Report Is Here

[woody woodpeckerbuzzing.zip](https://github.com/user-attachments/files/21008825/woody.woodpeckerbuzzing.zip)

**Comment by RafaelOliveira71594** [comment]
The New Update Didn't Fixed This Issue

**Comment by RafaelOliveira71594** [comment]
Anymore Ignored Me

**Comment by RafaelOliveira71594** [comment]
Is Not An ROM Issue It Does Not Buzz With SameBoy

**Comment by RafaelOliveira71594** [comment]
Please Fix Channel Audio 3 Soo We Can Fix That Buzzing Noise

**Comment by endrift** [comment]
I'll fix it when I have time. I said already that's after I release 0.11. Calm down and stop spamming the issue tracker.

**Comment by RafaelOliveira71594** [comment]
Thanks For That And Sorry For Spamming The Issue Tracker
